lib

```diff
@@ -1 +1 @@
-Subproject commit 8bab3f0ff1b879fb85366e45b31be8502c8fe551
+Subproject commit 6819993827720cc7faf880a5029cef013a59cae3
```

Comment:
I'm not 100% sure how this made it into the PR, much less how to revert it. Suggestions?